linode/manager
# feat: [M3-7948] - Add Region filtering to Linodes landing table

## Description 📝
Add support for region filtering in the Linodes landing table for Gecko LA. Region filter is saved in local storage similar to the "Show X" dropdown.

## Changes  🔄
List any change relevant to the reviewer.
- Add `site_type` to the linode instance type
- Removed `edge` references where possible
- Add region filter dropdown to the Linodes landing table

## Preview 📷
(ip address has been redacted)

https://github.com/user-attachments/assets/c538c7e2-6c46-415b-aedd-66683516cded

## How to test 🧪
### Verification steps
- `All` should display all Linodes (core and distributed regions)
- `Core` should only display Linodes in `core` regions
- `Distributed` should only display Linodes in `distributed` regions
- Ensure sorting, summary, and tag view still work as normal
- Navigate to another page and go back to the Landing page, the region selection should stay the same

## As an Author I have considered 🤔

*Check all that apply*
